kraken/src/api/handler/attacks/schema.rs

```diff
@@ -203,6 +203,22 @@ pub struct DnsResolutionRequest {
     pub workspace_uuid: Uuid,
 }
 
+/// Request to do DNS TXT scanning & parsing
+#[derive(Deserialize, ToSchema)]
+pub struct DnsTxtScanRequest {
+    /// The leech to use
+    ///
+    /// Leave empty to use a random leech
+    pub leech_uuid: Option<Uuid>,
+
+    /// The domains to resolve
+    #[schema(value_type = Vec<String>, example = json!(["example.com", "example.org"]))]
+    pub targets: Vec<String>,
```

Comment:
really necessary to declare the value_type? `Vec<String>` should be already parsable by utoipa

Reply:
so far it has been used in every `Vec<String>` field in this file like this - should I remove it from all of them?